index.bs

```diff
@@ -1903,6 +1916,16 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
     [=authenticators=] can be <a href="https://en.wikipedia.org/w/index.php?title=Hot_plug">hot-plugged</a> into (e.g., via USB)
     or discovered (e.g., via NFC or Bluetooth) by the [=client=] by various mechanisms, or permanently built into the [=client=].
 
+1. If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is present with the value
+     {{CredentialMediationRequirement/conditional}}:
+
+    1. If |conditionalCreateLifetimeTimer| is expired or |conditionalCreateOrigin| is not |callerOrigin|, throw a "{{NotAllowedError}}" {{DOMException}}.
+
+    Note: |conditionalCreateLifetimeTimer| and |conditionalCreateOrigin| will be set by the user agent after it believes an authentication ceremony has
+    been completed and the user consents to this type of credential creation.
+
+    1. Set |lifetimeTimer| to a client-specific default.
```

Comment:
This doesn't seem to take `pkOptions.timeout` into account at all, is that intended?

Reply:
In most cases, this will resolve quickly. Though I suppose clients might want to have optional authenticator selection upon conditional create one day. I modified this section to take in account `pkOptions.timeout`.